build_tools/pkgci/external_test_suite/gpu_rocm_models.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config_name": "gpu_rocm",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=rocm",
+      "--iree-rocm-target-chip=gfx90a",
```

Comment:
ROCm compilation is tied to a specific chip, so we should call out that chip (either by identifier like `gfx90a`, or by name like `MI250`). That should be reflected in both the `config_name` and the file name. If we run model tests on w7900, that will use a different chip (and thus config file).

Reply:
Makes sense, multiple machines have same chip name, so will go with the chip